gradle-suppressible-error-prone/src/main/java/com/palantir/gradle/suppressibleerrorprone/JdwpRemoteDebugPlugin.java

```diff
@@ -0,0 +1,59 @@
+/*
+ * (c) Copyright 2024 Palantir Technologies Inc. All rights reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.palantir.gradle.suppressibleerrorprone;
+
+import java.util.List;
+import org.gradle.api.Plugin;
+import org.gradle.api.Project;
+import org.gradle.api.tasks.compile.JavaCompile;
+import org.gradle.process.CommandLineArgumentProvider;
+
+/**
+ * To debug suppressible-error-prone in a build (works with breakpoints in VisitorStateModifications):
+ * 1. Run the "Debug errorprones" debug config which ships with this repository. Notably, it sets up a jdwp listener
+ *    at port 5006
+ * 2. Run the build with this plugin applied
+ */
+public abstract class JdwpRemoteDebugPlugin implements Plugin<Project> {
+    // No other way to programmatically check whether the configuration-cache is on
```

Comment:
not true! 

Use `BuildFeatures#getConfigurationCache`

Reply:
wow, thank you 🤯